### ISSUE TITLE
Which version to use?

### ISSUE THREAD

**Issue by ashikns** [aspnet/SignalR#2901]
It's difficult to understand the differences between various versions of the packages here. On nuget the latest released is 1.1.0 preview, the latest commit is to branch called 2.2, the master branch is 3.0(?). Which of the branches can I use? Also please note that I am planning to use Azure hosted SignalR and I read it's only supported in 2.2. Can anyone provide info what's the difference between these versions?

**Comment by davidfowl** [comment]
> On nuget the latest released is 1.1.0 preview.

Yes, 1.1.0 is the latest *preview*, you can use 1.0.3 if you want the latest stable bits.

> the latest commit is to branch called 2.2, the master branch is 3.0(?). 

It's confusing because we're following .NET Core and ASP.NET Core branch names. SignalR 1.1 is the 2.2 branch and 3.0 will be likely be the next version (still TBD)

> Also please note that I am planning to use Azure hosted SignalR and I read it's only supported in 2.2. Can anyone provide info what's the difference between these versions?

I don't know where you read that. It works with all released versions of SignalR. Can you point me to what you read so I can correct it?

**Comment by ashikns** [comment]
Thanks for clearing that up. As for the last bit, I saw this issue while browsing through: https://github.com/aspnet/SignalR/issues/2862. It had a label "2-working" that's where my assumption came from. However now I see that it also has a label "client:Java", so my bad.

One more thing: Do you guys have any changelogs between releases?

**Comment by davidfowl** [comment]
> One more thing: Do you guys have any changelogs between releases?

We're supposed to but we're been bad and haven't written anything since 2.1.0-preview2.

**Comment by ashikns** [comment]
Ah alright. I tried to figure it out from the commit history but the way branches were merged made it hard to follow them. I'll do a squash and compare them or so. Thanks again :)

**Comment by davidfowl** [comment]
@anurse @danroth27 Who does the release notes...?
